org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/DefaultIdeModule.xtend

```diff
@@ -20,4 +28,17 @@ class DefaultIdeModule extends AbstractGenericModule {
 		binder.bind(ExecutorService).toProvider(ExecutorServiceProvider)
 	}
 	
+	def void configureIResourceDescriptionsLiveScope(Binder binder) {
+		binder.bind(IResourceDescriptions)
+			.annotatedWith(Names.named(ResourceDescriptionsProvider.LIVE_SCOPE))
+			.to(LiveShadowedChunkedResourceDescriptions);
+	}
+	
+	def Class<? extends IContainer.Manager> bindIContainer$Manager() {
+		ProjectDescriptionBasedContainerManager
```

Comment:
can this break existing clients that expect SimpleResourceDescriptionsBasedContainerManager?

Reply:
Good catch. It delegates anyway. Removed binding